messages/tape_linux_sg/root.txt

```diff
@@ -37,7 +37,7 @@
 root:table {
 	messages:table {
 		start_id:int { 30200 }
-		end_id:int { 30399 }
+		end_id:int { 30499 }
```

Comment:
This is not good because the messages from `tape_linux_lin_tape` starts from `30400`.